src/scss/grommet-core/_objects.box.scss

```diff
@@ -507,16 +507,15 @@
   position: relative;
   max-width: 100%;
   width: 100vw;
+  height: 100vh;
+  max-height: 100%;
+  overflow: auto;
```

Comment:
Can we put the overflow only on "lap-and-up"? Just wanting to make sure we don't break anything with this change.